app/controllers/general.php

```diff
@@ -357,6 +358,15 @@ function router(App $utopia, Database $dbForConsole, callable $getProjectDB, Swo
     return false;
 }
 
+App::init()
+    ->groups(['database', 'function', 'storage', 'messaging'])
```

Comment:
Why are we being so specific on the groups and not using something like the API group?

Reply:
The console project needs access to some APIs (account/user) that are part of the `api` group, thought it would be easier to be specific so if any of these APIs need console access in future, we can just remove them instead of reworking the whole approach